e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js

```diff
@@ -0,0 +1,148 @@
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
+*/
+
+const webhookUtils = require('../../../utils/webhook_utils');
+
+let createdCommand;
+let simpleDialog;
+
+describe('ID17212 Interactive Dialog without element', () => {
+    before(() => {
+        // Set required ServiceSettings
+        const newSettings = {
+            ServiceSettings: {
+                AllowedUntrustedInternalConnections: 'localhost',
+                EnablePostUsernameOverride: true,
+                EnablePostIconOverride: true,
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Login as sysadmin and ensure that teammate name display setting us set to default 'username'
+        cy.apiLogin('sysadmin');
+        cy.apiSaveTeammateNameDisplayPreference('username');
+
+        // # Create new team and create command on it
+        cy.apiCreateTeam('test-team', 'Test Team').then((teamResponse) => {
+            const team = teamResponse.body;
+            cy.visit(`/${team.name}`);
+
+            const webhookBaseUrl = Cypress.env().webhookBaseUrl;
+
+            const command = {
+                auto_complete: false,
+                description: 'Test for simple dialog - no element',
+                display_name: 'Simple Dialog without element',
+                icon_url: '',
+                method: 'P',
+                team_id: team.id,
+                trigger: 'simple_dialog',
+                url: `${webhookBaseUrl}/simple_dialog_request`,
+                username: '',
+            };
+
+            cy.apiCreateCommand(command).then(({data}) => {
+                createdCommand = data;
+                simpleDialog = webhookUtils.getSimpleDialog(createdCommand.id, webhookBaseUrl);
+            });
+        });
+    });
+
+    it('UI check', () => {
+        // # Post a slash command
+        cy.postMessage(`/${createdCommand.trigger}`);
+
+        // * Verify that the interactive dialog modal open up
+        cy.get('#interactiveDialogModal').should('be.visible').within(() => {
+            // * Verify that the header of modal contains icon URL, title and close button
+            cy.get('.modal-header').should('be.visible').within(($elForm) => {
+                cy.get('#interactiveDialogIconUrl').should('be.visible').and('have.attr', 'src', simpleDialog.dialog.icon_url);
+                cy.get('#interactiveDialogModalLabel').should('be.visible').and('have.text', simpleDialog.dialog.title);
+                cy.wrap($elForm).find('button.close').should('be.visible').and('contain', '×').and('contain', 'Close');
+
+                cy.get('#interactiveDialogModalLabel').should('be.visible').and('have.text', simpleDialog.dialog.title);
```

Comment:
Is this line repeated intentionally? Looks like a duplicate of line 70.

Reply:
Oh no, done removing this line